cmd/cloud-api-adaptor/main.go

```diff
@@ -46,8 +48,13 @@ var hypcfg hypervisor.Config
 func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 
 	if len(os.Args) < 2 {
-		fmt.Printf("%s aws|azure|ibmcloud|libvirt <options>\n", os.Args[0])
+		fmt.Printf("%s aws|azure|ibmcloud|libvirt|vsphere <options>\n", os.Args[0])
 		cmd.Exit(1)
+	} else if len(os.Args) == 2 {
```

Comment:
I think a better option will be something like this:
```
if len(os.Args) < 2 || !contains(os.Args[1], supportedProviders)  {
...
}
```
wdyt ?

Reply:
@bpradipt, this is better. Changed it. PTAL